.env

```diff
@@ -32,8 +32,12 @@ TEST_LOG_REQUESTS=false
 # Default admin credentials - ALSO OVERRIDE IN PRODUCTION
 ADMIN_EMAIL=ni@aefeup.pt
 ADMIN_PASSWORD=n1j0bs_ftw.12345
+
 #CORS allowed origin - OVERRIDE IN PRODUCTION
 ACCESS_CONTROL_ALLOW_ORIGIN= 
```

Comment:
Why do we need both settings? We can just use `ACCESS_CONTROL_ALLOW_ORIGIN_REGEX_LIST` now.

Reply:
We don't, forgot to remove it from there